tools/ota_e2e_tests/aws_ota_test/aws_ota_project.py

```diff
@@ -219,7 +222,7 @@ def setClientCredentialsForAwsIotEndpoint(self, awsIotEndpoint, awsIotEndpointPo
         """
         self.__setIdentifierInFile(
             {
-                'static const char clientcredentialMQTT_BROKER_ENDPOINT[] =': '\"' + awsIotEndpoint + '\";',
+                '#define clientcredentialMQTT_BROKER_ENDPOINT ': '\"' + awsIotEndpoint + '\"',
```

Comment:
__setIdentifierInFile() already inserts a space after the key, although I think it does not hurt to have one here.

Reply:
Will remove it in the next commit to keep the consistency :-)